site/source/tutorials/accessibility.md

```diff
@@ -0,0 +1,250 @@
+
+##### Keyboard Interaction
+- **Left arrow**: Moves to previous tab button, wrapping from first to last
+- **Right arrow**: Moves to next tab button, wrapping from last to first
+- **Down arrow**: Moves to next tab button only on vertically oriented tabs
+- **Up arrow**: Moves to previous tab button only on vertically oriented tabs
+- **Page Up**: Moves to first tab button
+- **Page Down**: Moves to last tab button
+- **Escape**: Closes focused tab button if it is closeable
+
+___
+
+### TitlePane
+
+TitlePane creates an accordion widget, which is functionally a button that expands a content container. Collapsed content is obscured with `aria-hidden`, and the button and content container are associated with `aria-controls` and `aria-labelledby`.
+
+##### A11y properties
+- `headingLevel`: Optionally customize the heading level of the button controlling the accordion.
+___
+
+This overview only touched on properties that primarily exist for accessibility-related reasons, but any design decision or property change will end up affecting accessibility in some way. For some of those, such as `invalid`, `disabled`, and `readOnly`, we handle setting ARIA attributes in the background in addition to toggling classes without any extra attention needed from the widget author. In other cases, such as `getResultLabel` in ComboBox or `renderMonthLabel` in Calendar, it is entirely up to the author to ensure the returned result includes clear, screen-reader-accessible text content.
+
+## Styling
+The base Dojo 2 theme meets WCAG AA color contrast guidelines, but it is up to the widget author to double check the final styles as changes to font size or background could affect readability. Widget CSS packaged with Dojo2 also includes `:focus` styles for all focusable nodes.
+
+There is a set of base styles provided by `@dojo/widgets` separate from themes, containing basic utility classes like `.visuallyHidden`, which will make content invisible to sighted users but still allow it to be read by screen readers. To use it, import `baseCss` separately like so:
+
+```js
+import * as css from ‘path/to/your/css’;
+import * as baseCss from ‘@dojo/widgets/common/styles/base.m.css’;
+
+@theme(css)
+class MyWidget extends WidgetBase {
+	render() {
+		return v(‘div’, {
+			classes: this.classes().fixed(baseCss.visuallyHidden)
+		}, [ ‘Screen reader instructions’ ]);
+	}
+}
+```
+
+## Focus Management
+Dojo 2 will provide a focus manager for situations where focus needs to be directly managed, since the [virtual DOM approach](./working_with_virtual_dom) means directly touching the DOM within widgets is discouraged. This feature is currently in development.
+
+## Writing Custom Widgets
+All DOM attributes can be set with `VirtualDomProperties` so, apart from managing focus, no extra tools should be required to create strongly accessible widgets. For example, the following code would create an accessible popup button:
```

Comment:
Should probably also link to the tutorial on writing custom widgets in general, and how this is specific to a11y for custom widgets, and probably also look at the custom widgets tutorial and add an aside linking to this article